### PR TITLE
ci/blockers: fix a few CI blockers

### PR DESCRIPTION
**Pull request by yf13**
## Summary
This fixes two CI issues from `Linux(riscv)`;
It also enlarges timeout to fix LTP issues from `Linux(sim-01)`.

## Impact
ci check

## Testing
CI checks

### CODE REVIEW

File: arch/risc-v/src/common/espressif/esp_twai.c [hunk 1]
```diff
@@ -252,6 +252,7 @@ static void esp_twai_reset(struct can_dev_s *dev)
 
   ret = twai_hal_init(&priv->ctx, &hal_config);
   assert(ret);
```
Comment (xiaoxiang781216) [inline arch/risc-v/src/common/espressif/esp_twai.c:254]:
```suggestion
  ASSERT(ret);
```
and revert the change at line 255

**Review by xiaoxiang781216 — APPROVED** [review]

**Review by xiaoxiang781216 — APPROVED** [review]

**Review by xiaoxiang781216 — APPROVED** [review]

File: tools/ci/testrun/script/test_open_posix/test_openposix_.py [hunk 2]
```diff
@@ -2026,7 +2026,7 @@ def test_ltp_interfaces_sigwait_6_1(p):
 
 def test_ltp_behavior_WIFEXITED_1_2(p):
     ret = p.sendCommand(
-        "ltp_behavior_WIFEXITED_1_2", ["PASSED", "passed", "Passed", "PASS"], timeout=10
+        "ltp_behavior_WIFEXITED_1_2", ["PASSED", "passed", "Passed", "PASS"], timeout=40
```
Comment (xiaoxiang781216) [inline tools/ci/testrun/script/test_open_posix/test_openposix_.py:2029]: Extending timeout can't fix the ci failure, so we drop this change?

Comment (yf13) [reply tools/ci/testrun/script/test_open_posix/test_openposix_.py]: Yes, I dropped two changes but kept the working one.

**Review by jerpelea — APPROVED** [review]

**Review by xiaoxiang781216 — APPROVED** [review]
